resources/resource_definitions.py

```diff
@@ -458,8 +458,7 @@ def put(self, id, item_id):
                     item.name = json_data['name']
 
                 if 'done' in json_data:
-                    if json_data['done'] == "True":
-                        item.done = json_data['done']
+                    item.done = json_data['done']
```

Comment:
**Code Quality: Nitpick**

This way I can set done to anything. Are there no checks to ensure that the value of `done` is either True or False?